src/components/Button/Button.module.css

```diff
@@ -4,8 +4,8 @@
   font-weight: 800;
   min-height: 35px;
   min-width: 40px;
-  border: 1px solid #3f3feb;
-  color: #3f3feb;
+  border: 1px solid #000;
+  color: #000;
```

Comment:
todo: I recommend you to start using CSS variables as a way to group/reuse all the colors used in this app :)